src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -146,9 +149,7 @@ class ActiveFocusInvestigation extends React.Component<
 
   public componentDidMount() {
     const { userName, fetchPlansActionCreator, supersetService } = this.props;
-    const supersetParams = superset.getFormData(2000, [
-      { comparator: InterventionType.FI, operator: '==', subject: 'plan_intervention_type' },
-    ]);
+    const supersetParams = superset.getFormData(2000, supersetFIPlansParamFilters);
```

Comment:
Use the `SUPERSET_MAX_RECORDS` env instead of `2000` for number of records to be fetched.